explore-analyze/visualize/charts/bar-charts.md

```diff
@@ -0,0 +1,393 @@
+    - **Show gridlines**: Toggle gridlines on or off for better readability.
+    % Show gridlines is not an option in the right menu, but is in the Left axis menu.
+
+### Horizontal axis settings [horizontal-axis-options]
+
+**Data**
+:   The dimension that creates your individual bars. Common options include:
+    - **Terms**: Show top values from a categorical field (like product names, regions, or user types)
+    % Terms is not an option.
+    - **Date histogram**: Create time-based bars with configurable intervals
+    - **Histogram**: Create numeric ranges or buckets
+    % Histogram is no an option.
+    - **Filters**: Define custom categories using KQL queries
+    - **Intervals**: ?
+    - **Top values**: ?
+
+**Appearance**
+:   Define the formatting of the horizontal axis, including:
+    - **Name**: Customize the axis label to describe what the bars represent.
+    % None of the following options are available in the UI.
+    - **Number of values**: For terms aggregations, control how many bars to show (displays top N values by the selected metric).
+    - **Sort by**: Select what determines the order of bars—by the metric value, alphabetically, or by document count.
+    - **Sort direction**: Ascending or descending order.
+    - **Show labels**: Toggle category labels on or off.
+    - **Label rotation**: Rotate labels to prevent overlap when you have many categories or long names.
+    - **Show gridlines**: Toggle vertical gridlines on or off.
+
+### Break down by settings [breakdown-options]
+
+**Data**
+:   Split your bars into segments or groups based on another dimension. Each unique value creates its own segment or bar, allowing you to show composition or compare metrics across multiple dimensions.
+
+    - **Number of values**: The number of segments or groups to show. If more values are available, they can be grouped as **Other**.
+    - **Rank by**: The dimension by which values are ranked.
+    - **Rank direction**: The direction to use for the ranking.
+    - **Collapse by**: Exclude this dimension from the visualization. All metric values sharing the same value for this dimension should be aggregated into a single number.
+
+    :::{include} ../../_snippets/lens-breakdown-advanced-settings.md
+    :::
+
+**Appearance**
+:   Define the formatting of the breakdown, including:
+    - **Name**: Customize the legend label.
+    - **Colors**: Select a color palette or assign specific colors to categories.
+    - **Show legend**: Toggle the legend display on or off.
+    % Show legends not an option
+    - **Legend position**: Place the legend on the right, top, or bottom of the chart.
+    % Legend position not an option
+
+### General chart settings [appearance-options]
+
+When creating or editing a visualization, you can customize several appearance options. Look for the {icon}`brush` icon to access these settings.
+
+**Layout**
+% This is not available from the menu that you access when clicking the brush icon.
```

Comment:
This comment is affecting the list rendering. Will need to remove once it's addressed.